packages/chart-advisor/src/types/spec.ts

```diff
@@ -0,0 +1,9 @@
+import type { AntVSpec, ChartAntVSpec, GraphAntVSpec } from '@antv/antv-spec';
+
+/**
+ * @public
+ */
+export type Specification = AntVSpec;
+
+export type ChartSpec = ChartAntVSpec;
```

Comment:
Why is there a blank line between 6 and 7

Reply:
that's ... fine